extensions/search-router/package.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "search-router",
   "title": "Search Router",
-  "description": "Local-first search with DuckDuckGo !Bangs (similar to unduck.link)",
+  "description": "Local-first search with Kagi's !Bangs (similar to unduck.link)",
```

Comment:
We still want the extension to be searchable for the DuckDuckGo term in the store. As long as having that in the keywords array accomplishes this, we should be good with changing the description her. @pernielsentikaer any comment?

Reply:
what about "Local-first search with Kagi's !Bangs (similar to unduck.link with DuckDuckGo's !Bangs)"?

Reply:
💪 Go for it!